src/components/Profile/Profile.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+export default function Profile() {
+  return (
+    <React.Fragment>
+      <div class='wrapper row'>
+        <div class='col-lg-4 col-md-8 col-sm-10'>
```

Comment:
```suggestion
        <div class='col-lg-4 col-md-8 col-sm-10 text-center'>
```
I suggest to center it, but its just taste :)